docs/topics/clusterdefinitions.md

```diff
@@ -36,7 +36,7 @@ $ aks-engine get-versions
 
 | Name                            | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                   |
 | ------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| addons                          | no       | Configure various Kubernetes addons configuration (currently supported: tiller, kubernetes-dashboard). See `addons` configuration below                                                                                                                                                                                                                                                                       |
+| addons                          | no       | Configure various Kubernetes addons configuration. See `addons` configuration [below](#addons)                                                                                                                                                                                                                                                                       |
```

Comment:
not sure this link works, you might have to add `<a name="feat-addons"></a>` above the title

Reply:
verified it works (I clicked through the "view file" link)